### PR TITLE
feat(images): migrate images from `site/content/en/a*` to storage bucket

### PR DESCRIPTION
**Pull request by MichaelSolati**

### CODE REVIEW

File: src/site/content/en/accessible/headings-and-landmarks/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ own CSS, and don't skip levels.
 For example, this site has a section called "IN THE NEWS", followed by two
 headlines:
 
-<img class="screenshot" src="./headings.png" alt="A news site with a headline, hero image, and subsections.">
+{% Img src="image/admin/CdBjBuUo2yVVHWVFnQzx.png", alt="A news site with a headline, hero image, and subsections.", width="800", height="414", class="screenshot" %}
```
Comment (robdodson) [inline src/site/content/en/accessible/headings-and-landmarks/index.md:44]:
```suggestion
{% Img src="image/admin/CdBjBuUo2yVVHWVFnQzx.png", alt="A news site with a headline, hero image, and subsections.", width="800", height="414", class="w-screenshot" %}
```

File: src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml [hunk 2]
```diff
@@ -3,10 +3,7 @@ setLeader: |
 height: unset
 tabLabel: sample
 questions:
-  - stimulus: |
-      ```html
-      <img src="temp.svg" alt="temp">
-      ```
+  - stimulus: "![temp](image/user/temp.svg)"
```
Comment (robdodson) [inline src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml:6]: I think this was changed by mistake

File: src/site/content/en/animations/animations-guide/index.md [hunk 3]
```diff
@@ -273,19 +269,15 @@ If you see entries for [**Recalculate Style**](https://developer.mozilla.org/en-
 then the browser is having to begin at the start of the [rendering waterfall](https://developer.mozilla.org/en-US/docs/Tools/Performance/Scenarios/Animating_CSS_properties).
 
 <figure>
-  <img class="w-screenshot w-screenshot--filled"
-       src="waterfall-before.jpg"
-       alt="The Waterfall panel shows many entries for Recalculate Style.">
+  ![The Waterfall panel shows many entries for Recalculate Style.](waterfall-before.jpg), class="w-screenshot w-screenshot--filled" %}
```
Comment (robdodson) [inline src/site/content/en/animations/animations-guide/index.md:272]: looks like this one was missed

File: src/site/content/en/animations/animations-guide/index.md [hunk 4]
```diff
@@ -273,19 +269,15 @@ If you see entries for [**Recalculate Style**](https://developer.mozilla.org/en-
...
     The <a href="https://animation-with-top-left.glitch.me/">animation-with-top-left</a>
     example causes style recalculation.
   </figcaption>
 </figure>
 
 <figure>
-  <img class="w-screenshot w-screenshot--filled"
-       src="waterfall-after.jpg"
-       alt="The Waterfall panel shows no entries for Recalculate Style.">
+  ![The Waterfall panel shows no entries for Recalculate Style.](waterfall-after.jpg), class="w-screenshot w-screenshot--filled" %}
```
Comment (robdodson) [inline src/site/content/en/animations/animations-guide/index.md:280]: looks like this one was missed

**Review by robdodson — APPROVED** [review]
LGTM with comments

File: src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml (hunk 2 above)
Comment (MichaelSolati) [reply src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml:6]: This was done on purpose, I want to push authors to use markdown for the stimulus image as we have a mechanism in the assessment component to extract the markdown syntax to use the cdn.

**Pull request merged**